### PR TITLE
labeler: add agps GI label

### PR DESCRIPTION
**Pull request by richardmccrae-nordicsemi**
added agps GI_LABEL as well as all associated files

Signed-off-by: Richard McCrae <richard.mccrae@nordicsemi.no>

### CODE REVIEW

**Review by wiminordic — APPROVED** [review]

**Review by guwa — APPROVED** [review]

File: .github/labeler.yml [hunk 1]
```diff
@@ -18,6 +18,16 @@
...
 
+"CI-agps-test":
+  - "samples/nrf9160/gps/*"
+  - "samples/nrf9160/agps/*"
+  - "include/net/cloud.h"
+  - "include/net/nrf_cloud.h"
+  - "include/net/nrf_cloud_agps.h"
+  - "include/net/nrf_cloud_pgps.h"
+  - "include/lib/agps/*"
+  - "include/lib/supl/*"
```
Comment (tokangas) [inline .github/labeler.yml:29]: These should probably be without `include`?

```suggestion
  - "lib/agps/*"
  - "lib/supl/*"
```
And probably would be good to add `include/modem/agps.h` as well.

**Review by tokangas — CHANGES_REQUESTED** [review]

File: .github/labeler.yml (hunk 1 above)
Comment (richardmccrae-nordicsemi) [reply .github/labeler.yml]: Good catch

Comment (tokangas) [reply .github/labeler.yml]: What about adding `include/modem/agps.h`? I think that should be included too.

**Review by tokangas — APPROVED** [review]

**Review by thst-nordic — APPROVED** [review]

**Review by miha-nordic — APPROVED** [review]

**Comment by richardmccrae-nordicsemi** [comment]
Need to wait on this one until AGPS test repo takes into consideration new GNSS sample

**Comment by richardmccrae-nordicsemi** [comment]
no longer needed

**Pull request closed without merging**